client/src/components/dashboard/Dashboard.js

```diff
@@ -1,15 +1,39 @@
 import React from 'react';
+import DashboardSummary from './DashboardSummary';
+import EntryList from '../entries/EntryList';
+
+// Dummy Data
+import entries from '../../data/entries';
+
+const initialState = {
```

Comment:
Instead of using dummy data, should create methods to calculate `monthlyAmountSaved`, etc, using the `entries`